code/lib/preview-api/src/modules/store/csf/prepareStory.ts

```diff
@@ -149,6 +154,25 @@ export function prepareStory<TRenderer extends Renderer>(
   };
 }
 
+export const getTriangularNumber = (seed: number) => (seed * (seed + 1)) / 2;
```

Comment:
`getTriangularNumber` seems like a very basic math utility. Fairly sure there's a place I should import it from, but it's also minimal. Let me know where it might properly belong if you'd like it moved.